x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/profiling/10_basic.yml

```diff
@@ -0,0 +1,177 @@
+---
+setup:
+  - skip:
+      version: " - 8.12.99"
```

Comment:
I don't know the specs of the version string here, but looks like it means 8.13+ (if it follows the common .99 use cases).

Reply:
Yes, that's correct. The established practice is to specify the restriction like that.